### PR TITLE
zh: refine TiDB-Binlog relay log

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

refine TiDB-Binlog relay log

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@seiya-annie @july2993 PTAL

**Review by july2993 — APPROVED** [review]

File: tidb-binlog/tidb-binlog-relay-log.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Drainer 开启 relay log 后会先将 binlog event 写到磁盘上，然后再
 
 ### Relay log 的清理（GC）机制
 
-Drainer 在运行时，如果确认已经将一个 relay log 文件的全部数据都成功同步到下游了，就会马上删除这个文件，所以 relay log 不会占用过多空间。一个 relay log 文件大小达到 10MB（默认）时就会做切分，数据开始写入新的 relay log 文件。
+Drainer 在将数据同步到下游之前，会先将数据写入到 relay log 文件中，当一个 relay log 文件大小达到 10MB（默认）后就会开始将数据写入到下一个 relay log 文件中。当 drainer 将数据成功同步到下游后，就会自动清理除当前正在写入的 relay log 文件外的其他已完成同步的 relay log 文件。
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-relay-log.md:41]:
```suggestion
Drainer 在将数据同步到下游之前，会先将数据写入到 relay log 文件中。当一个 relay log 文件大小达到 10MB（默认）后，Drainer 就会开始将数据写入到下一个 relay log 文件中。当 Drainer 将数据成功同步到下游后，就会自动清除当前正在写入的 relay log 文件以外其他已完成同步的 relay log 文件。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by seiya-annie — COMMENTED** [review]
当一个relay log文件大小大于等于10M（默认）并且最后一个事务被完整写入后，Drainer 就会开始将数据写入到下一个 relay log 文件中。

**Review by seiya-annie — APPROVED** [review]

**Pull request merged**